src/index.ts

```diff
@@ -47,89 +136,53 @@ export class I18N {
     }
 
     i18n(keysetName: string, key: string, params?: Params): string {
-        const languageData = this.getLanguageData(this.lang);
-
-        if (typeof languageData === 'undefined') {
-            throw new Error(`Language '${this.lang}' is not defined, make sure you call setLang for the same language you called registerKeysets for!`);
+        if (!this.lang && !this.fallbackLang) {
+            throw new Error('There are no available languages. You should set at least one of these languages: lang, fallbackLang');
```

Comment:
```suggestion
            throw new Error('Language is not set. You should set at least one of these: "lang", "fallbackLang"');
```

Also same error should be while using method `has`. So it worth extracting this code into something like `ensureLang()`